apps/pie-docs/src/components/toast/overview/overview.md

```diff
@@ -0,0 +1,376 @@
+    ]
+} %}
+
+---
+
+## Variations
+
+### Default
+
+A default one-line toast, this can be used with or without an icon or action.
+
+{% contentPageImage {
+    src:"../../../assets/img/components/toast/variations-default.svg",
+    alt: "A toast component on its default variation.",
+    width: 300
+} %}
+
+### Multi-line
+
+If you require additional text for your message, please use the multi-line component. Maximum of 3 lines.
+
+{% contentPageImage {
+    src:"../../../assets/img/components/toast/variations-multi-line.svg",
+    alt: "A toast component on its multi-line variation.",
+    width: 300
+} %}
+
+---
+
+## Modifiers
+
+### Strong
+
+To create a stronger impact with your message, you can use strong variant. This uses colour to back-up your message
+
+{% contentPageImage {
+    src:"../../../assets/img/components/toast/modifiers-strong.svg",
+    alt: "Multiple toasts with strong modifier.",
+    width: 866
+} %}
+
+{% list {
+    type: listTypes.ordered,
+    items: [
+        "**Info:** Providing contextual information about the progress of an action",
+        "**Warning:** Providing non-critical information about the progress of an action",
+        "**Success:** Providing positive information about the progress of an action",
+        "**Error:** Providing critical information about the progress of an action. Please review if Toast is correct to use it might be better to use a dialogue, notification or banner"
+    ]
+} %}
+
+### Subtle
+
+To create a message with less prominence you can use the subtle variant.
+
+{% contentPageImage {
+    src:"../../../assets/img/components/toast/modifiers-subtle.svg",
+    alt: "Multiple toasts with strong modifier.",
+    width: 788
+} %}
+
+{% list {
+    type: listTypes.ordered,
+    items: [
+        "**Info:** Providing contextual information about the progress of an action",
+        "**Warning:** Providing non-critical information about the progress of an action",
+        "**Success:** Providing positive information about the progress of an action",
+        "**Error:** Providing critical information about the progress of an action. Please review if Toast is correct to use it might be better to use a dialogue, notification or banner",
+        "**Neutral:** Providing critical information about the progress of an action. Please review if Toast is correct to use it might be better to use a dialogue, notification or banner"
+    ]
+} %}
+
+### Icons
+
+When incorporating an icon into a Toast, it is essential to ensure that the icon clearly conveys the intended feedback. You should only use the icons built into the component and not use a different icon.
+
+{% usage {
+    do: {
+        type: usageTypes.image,
+        items: [{
+            src: "../../../assets/img/components/toast/modifiers-icons-do.svg",
+            width: "456px",
+            alt: "Two toast components with close icon button"
```

Comment:
I don't think the close button is relevant for this section
```suggestion
            alt: "Two toast components using approved icons for the error and info states."
```

Reply:
done